typescript/infra/config/environments/mainnet/index.ts

```diff
@@ -18,7 +18,7 @@ export const environment: CoreEnvironmentConfig<MainnetChains> = {
   environment: environmentName,
```

Comment:
I can't comment on it because it's just a rename, but the renames of `mainnet/helloworld/abacus/` -> `mainnet/helloworld/hyperlane/` (and the same for testnet2) I think will break some things because we're leaving the context name as `'abacus'`. The file path is crafted using the context name

Reply:
Noted, will revert, thanks!